pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -265,6 +265,19 @@ func (r *ReconcilePerconaServerMongoDB) updateConfigMembers(ctx context.Context,
 			Votes:        mongo.DefaultVotes,
 		}
 
+		if len(rs.Horizons) > 0 {
+			horizons := make(map[string]string)
+			for h, domain := range rs.Horizons[pod.Name] {
+				d := domain
+				if !strings.HasSuffix(d, ":27017") {
```

Comment:
Maybe use `DefaultMongoDBPort`.

Reply:
I would but its a int32 and I need to convert it to string and this place will be very messy.

Reply:
This "horizons" feature should allow you to chose ports, and not force the exclusive use of port 27017.